lib_dsp/lib_build_info.cmake

```
@@ -1,7 +1,7 @@
 set(LIB_NAME lib_dsp)
-set(LIB_VERSION 6.2.1)
+set(LIB_VERSION 6.3.0)
 set(LIB_INCLUDES api)
-set(LIB_DEPENDENT_MODULES lib_logging)
+set(LIB_DEPENDENT_MODULES lib_logging(3.2.0))
```

```suggestion
set(LIB_DEPENDENT_MODULES "lib_logging(3.2.0)")
```

It needs to be quoted like this to allow cmake to process it correctly.